pulser-core/pulser/backend/results.py

```diff
@@ -142,6 +145,37 @@ def _find_uuid(self, observable: Observable | str) -> uuid.UUID:
                 "nor a known observable tag in the results."
             )
 
+    def _to_abstract_repr(self) -> dict:
+        d = deepcopy(self.__dict__)
```

Comment:
This is a bit hacky and we might get all sorts of unwanted stuff. How about just this?
```suggestion
        d = {"atom_order": self.atom_order, "total_duration": self.total_duration}
```